scripts/common.py

```diff
@@ -476,6 +474,7 @@ def transpile(cc_db_path: str,
     """
     run the transpiler on all C files in a compile commands database.
     """
+    assert config.C2RUST_BIN is not None
```

Comment:
Can you add similar `# for mypy` comments where you use these assertions specifically for `mypy` like in one of the earlier ones?